io/zenoh-transport/src/unicast/establishment/open.rs

```diff
@@ -569,13 +569,16 @@ pub(crate) async fn open_link(
         ext_compression: ext::compression::CompressionFsm::new(),
     };
 
+    #[allow(clippy::unnecessary_min_or_max)]
```

Comment:
```suggestion
    // Clippy raises a warning because `batch_size::UNICAST` is currently equal to `BatchSize::MAX`.  
    // However, the current code catches the cases where `batch_size::UNICAST` is different from `BatchSize::MAX`.
    #[allow(clippy::unnecessary_min_or_max)]
```